src/pages/calendar.tsx

```diff
@@ -0,0 +1,38 @@
+import { Calendar as ReactCalendar } from 'react-calendar'
+import 'react-calendar/dist/Calendar.css';
+
+import { Container, Text } from "@chakra-ui/react";
+
+import { useState } from 'react';
+
+type ValuePiece = Date | null;
+
+type Value = ValuePiece | [ValuePiece, ValuePiece];
+
+const Calendar = () => {
+    const [value, onChange] = useState<Value>(new Date());
+    const [clickedDate, setClickedDate] = useState<ValuePiece>(null);
+
+    // tileContent: custom content within item

```

Comment:
I would remove these comments until we're ready to add the custom content

Reply:
removed